packages/suite/src/views/wallet/staking/components/EthStakingDashboard/components/StakingCard.tsx

```diff
@@ -75,6 +66,12 @@ const StyledButton = styled(Button).attrs(props => ({
     font-size: ${variables.FONT_SIZE.NORMAL};
 `;
 
+const Wrapper = styled.span`
+    display: flex;
+    gap: ${spacingsPx.xxs};
+    align-items: center;
+`;
+
```

Comment:
We have component for layout use cases like this one:

```tsx
<Row gap={spacings.xxs} alignItems="center">...</Row>
```

Would it fit to your use case?

Reply:
<img width="400" alt="Screenshot 2024-08-21 at 14 24 53" src="https://github.com/user-attachments/assets/04a39b84-095d-4505-8cdf-f677d3bb95ee">
fixed, look the same, merging